hydra-node/hydra-node.cabal

```diff
@@ -305,6 +305,7 @@ test-suite tests
     , base
     , base16-bytestring
     , bytestring
+    , cardano-api
```

Comment:
Must: not depend on cardano-api directly

Everything is re-exported and extended through hydra-cardano-api

Reply:
This is not needed in the end.